### ISSUE TITLE
New Linux Compile Error (v9.3.0)

### ISSUE THREAD

**Issue by Joykiller** [chaincoin-legacy/chaincoin#54]
2017-08-09 11:42:00 Using wallet wallet.dat
2017-08-09 11:42:00 init message: Verifying wallet...
2017-08-09 11:42:00 CDBEnv::Open : LogDir=/home/***/.chaincoin/database ErrorFile=/home/***/.chaincoin/db.log
2017-08-09 11:42:00 ERROR: CDBEnv::Open : Error 22 opening database environment: Invalid argument

2017-08-09 11:42:00 Moved old /home/***/.chaincoin/database to /home/***/.chaincoin/database.1502278920.bak. Retrying.
2017-08-09 11:42:00 CDBEnv::Open : LogDir=/home/***/.chaincoin/database ErrorFile=/home/***/.chaincoin/db.log
2017-08-09 11:42:00 ERROR: CDBEnv::Open : Error 22 opening database environment: Invalid argument

Got any ideas? Berkeley was 4.8.30 - even reinstalled to make sure it wasnt it.
./configure --with-gui=qt4 LDFLAGS="-L/usr/local/BerkeleyDB.4.8/lib/" CPPFLAGS="-I/usr/local/BerkeleyDB.4.8/include/"

above flags used.

**Comment by brakmic** [comment]
The root of the problem could be in many places. However, may I ask you to use [this tutorial](https://github.com/chaincoin/chaincoin/blob/master/doc/build-unix.md) to compile with a separately downloaded BerkeleyDB?

Follow the tutorial by setting the needed paths first, CHC_ROOT and BDB_PREFIX, manually download the BDB sources and build the library. Finally configure & compile chaincoin binaries.

**Comment by Joykiller** [comment]
I've already done that. CHC_ROOT and BDB_PREFIX just tell the locations of the DB install paths for the lib/includes. I've installed a new copy set as mentioned above and set the location flags as the script does. My other bitcoin based wallets work, just not this one. 

I even just redownloaded and tried again with the above file:  ~/test/db-4.8.30.NC/build_unix $ etc.

Not sure what else it would be.

PS: You have a deprecated INCLUDES on line 733 in configuration.ac 

Nevermind, I fixed it, derp'ed out and forgot to make clean on other attempts from last night when I was recompiling with the previously downloaded DB.
